example-get-started/code/src/featurization.py

```diff
@@ -56,7 +56,7 @@ def generate_and_save_train_features(train_input, train_output, bag_of_words, tf
         tfidf (sklearn.feature_extraction.text.TfidfTransformer): TF-IDF transformer.
     """
     df_train = get_df(train_input)
-    train_words = np.array(df_train.text.str.lower().values.astype("U"))
+    train_words = np.array(df_train.text.str.lower().values)
```

Comment:
curious - why this change?

Reply:
It was breaking (just terminating the process without any hints). Not sure what was the problem. And IIUC, `.astype("U")` convert value to string, but it's already string. So not sure why it was needed in the first place :)

Reply:
@aguschin can we check blame please and see who put it there in the first place and ask for the review? (can be me of course :) just in case if @daavoo and @dberenbaum remember something)

Reply:
This goes all the way back to the creation of this repo 5 years ago: https://github.com/iterative/example-repos-dev/blame/e22378ce6a7c1729d3d5bdfe365be38bad605353/code/featurization.py

it could be python 2 (for which this operation made sense IIUC)

don't think it's relevant to double-check this :)